modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
+//provides includes for reference sections in anypoint-insights.adoc 
+
+//-----------------------//
+// APP DETAIL HEADER INFO:
+//tag::app-detail-header[]
+Type::
+Mule application type (*Application*)
+
+Status::
+Status code that indicates whether this Mule app is running or in some other xref:runtime-manager::managing-deployed-applications.adoc#status_states[state]
++
+This code comes from Runtime Manager.
+
+URL:: 
+URL of this deployed Mule app
+
+//TODO: Runtime -> Runtime Version (SATHISH)
+Runtime::
+Mule runtime engine version of this Mule app
+
+Replicas::
+Virtual core fraction and count for this Mule app, for example, *0.1 vCores x 1* (= 10% of 1 vCore)
+
+Last Updated::
+Date-time stamp for the last redeployment of this Mule app
++
+This stamp comes from Runtime Manager data.
+//end::app-detail-header[]
+//
+//
+// APP DETAIL: Overview tab
+//tag::app-detail-overview[]
+
+//  shared metric in Overview tab
+include::partial$include-insights-charts.adoc[tag="average-latency"]
+include::partial$include-insights-charts.adoc[tag="p95-latency"]
+include::partial$include-insights-charts.adoc[tag="total-requests"]
+
+//TODO: Messages -> Message (SATHISH)
+Mule Messages Volume::
+Total Mule message count for this app
++
+For information about Mule messages, see xref:mule-runtime::about-mule-event.adoc[].
+
+//  shared metric in Overview tab
+include::partial$include-insights-charts.adoc[tag="request-latency"]
+
+Inbound Request Volume::
+Inbound request count at specific time points
```

Comment:
how about `Total number of inbound requests received by the application for the selected time duration`?

Reply:
I'm leaving out "for the selected time duration" because that is explained for every chart in the intro, and it would need to be added to every description, but it doesn't differentiate the chart data. And note that the totals for data points are collected at specific time points. 

```suggestion
Total number of inbound requests to this app at specific time points
```